lib/stack_master.rb

```diff
@@ -54,6 +54,18 @@
 module StackMaster
   extend self
 
+  def interactive?
+    !non_interactive?
```

Comment:
I'm not sure if defaulting to yes in that case would be expected behaviour though. Perhaps we could handle that case and output a message about the `-f` option.